### PR TITLE
use new Zbirka PDF instead of PNGs

### PR DESCRIPTION
**Pull request by ruthrootz**

### CODE REVIEW

File: PDFAnalyzer/docs.md
Comment (Nomalah) [inline PDFAnalyzer/docs.md:1]: Oh my goodness what a beauty of docs

Comment (ruthrootz) [reply PDFAnalyzer/docs.md:1]: Ikr? Pretty basic and mostly self-explanatory, but might help save someone time in the future.

**Comment by Nomalah** [comment]
Let me know when this is ready to go

**Comment by ruthrootz** [comment]
> Let me know when this is ready to go

Tested, ready to go.

File: public/books/ZG/songs.json [hunk 1]
```diff
@@ -1,1991 +1,2991 @@
 {
 	"1": {
 		"title": "U tamnoj noći",
-		"notes": ["E4", "C4", "G3", "C3"]
+		"notes": ["E4", "C4", "G3", "C3"],
+		"page_num": 0,
+		"starts_at_top": true
```
Comment (Nomalah) [inline public/books/ZG/songs.json:6]: Could you remove the page_num and starts_at_top keys for the production songs.json
```suggestion
		"notes": ["E4", "C4", "G3", "C3"]
```

**Review by Nomalah — CHANGES_REQUESTED** [review]
Could you remove the songs_old directory? Unless there's a purpose behind it?

Could you also crop the right hand side of the pdfs so that they're centered?
image

Otherwise looks great!

File: public/books/ZG/songs.json (hunk 1 above)
Comment (ruthrootz) [reply public/books/ZG/songs.json]: > Could you also remove the page_num and starts_at_top keys for the production songs.json

removed

**Comment by ruthrootz** [comment]
> Could you remove the songs_old directory? Unless there's a purpose behind it?
> 
> Could you also crop the right hand side of the pdfs so that they're centered? image
> 
> Otherwise looks great! P.S. You can add yourself under the credits in the readme

New crop. Let me know what you think!

**Review by Nomalah — APPROVED** [review]
LGTM

**Pull request merged**